basic-javascript/project-2.spec.js

```diff
@@ -1,3 +1,127 @@
 const funcs = require('./project-2');
 
-// whoops.. there is no test suite for this file. You'll simply just have to create one :/
+describe('getBiggest', () => {
+    test('6 or 7', () => {
+        const result = funcs.getBiggest(6,7);
+        expect(result).toBe(7)
+    });
+});
+
+describe('greeting', () => {
+    test('Norwegian', () => {
+        const result = funcs.greeting('Norwegian');
+        expect(result).toBe('Hello!');
+    });
+});
+
+describe('isTenOrFive', () => {
+    test('10is10', () => {
+        const result = funcs.isTenOrFive(10);
+        expect(result).toBe(true);
+    });
+});
+
+describe('isInRange', () => {
+    test('30IsInRange', () => {
+        const result = funcs.isInRange(30);
+        expect(result).toBe(true);
+    });
+});
+
+describe('isInteger', () => {
+    test('PiIsNot', () => {
+        const result = funcs.isInteger(Math.PI);
+        expect(result).toBe(false);
+    });
+});
+
+describe('fizzBuzz', () => {
+    test('15IsFizzbuzz', () => {
+        const result = funcs.fizzBuzz(15);
+        expect(result).toBe('fizzbuzz');
```

Comment:
Insufficient, there are at least four cases to test:
factors of 5
factors of 3
factors of 5 and 3
non-factors of 3 or 5